### PR TITLE
Add Plugin.uninstall callback support

### PR DESCRIPTION
**Pull request by wheaney**
https://github.com/SteamDeckHomebrew/decky-loader/issues/536

Please tick as appropriate:
- [x] I have tested this code on a steam deck or on a PC
- [x] My changes generate no new errors/warnings
- [x] This is a bugfix/hotfix
- [ ] This is a new feature

# Description

This fixes issue: #536

### CODE REVIEW

**Comment by wheaney** [comment]
Here is the PR for the plugin template update: https://github.com/SteamDeckHomebrew/decky-plugin-template/pull/34

**Comment by wheaney** [comment]
@TrainDoctor what's next for this?

**Comment by TrainDoctor** [comment]
> @TrainDoctor what's next for this?

Needs testing. @SteamDeckHomebrew/decky-testers please give this a shot and let me know if you encounter any issues.

**Comment by TrainDoctor** [comment]
@wheaney apologies for the delay. I finally have time to give this a whirl tomorrow. If you can rebase it to latest main that'd make it much easier for me to test.

**Comment by wheaney** [comment]
I'm not sure what was going on with that first merge, but I think it's all sync'ed up now.

File: contrib/deck.sh
Comment (JohnnyCrazy) [inline contrib/deck.sh:1]: Empty file - should be removed

**Review by JohnnyCrazy — CHANGES_REQUESTED** [review]
I think this is a good addition - can you remove the empty file and rebase? Will do some testing the next days.

**Comment by TrainDoctor** [comment]
> I think this is a good addition - can you remove the empty file and rebase? Will do some testing the next days.

@wheaney once this change is made I can pretty easily merge this. Thanks.

**Comment by wheaney** [comment]
The commit history here was confusing me, so I just rebased, removed the empty file, and force pushed.

**Comment by JohnnyCrazy** [comment]
Mergin, thanks for this @wheaney !

**Pull request merged**